spec/rails_helper.rb

```diff
@@ -5,7 +5,6 @@
 
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
-require "spec_helper"
```

Comment:
Why has this been removed?

Reply:
it's already being required up at the top - think it was added by accident in the past